src/main/webapp/script.js

```diff
@@ -87,6 +87,18 @@ function addQueryParameter(key, value){
   window.history.pushState({path: newURL},'',newURL);
 }
 
+function logAddressInput(){
+  let streetNumber = document.getElementById('street_number').value;
+  let route = document.getElementById('route').value;
+  let city = document.getElementById('locality').value;
+  let state = document.getElementById('administrative_area_level_1').value;
+  let zipCode = document.getElementById('postal_code').value;
+  let country = document.getElementById('country').value;
+
+  addQueryParameter("address", streetNumber + " " + route + " " + city + " "
```

Comment:
Please use template literals
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I didn't know this was an option -- thanks for linking! Changing right now.

Reply:
@julian-yang need your approval to push, thanks!